Deploy/stacks/dynamic/stack-data-uploader/README.md

```diff
@@ -272,6 +272,14 @@ The icons can be found at `GEOSERVER_URL/www/icons` and the "other files" (being
 }
 ```
 
+Note: If you want to reference icons uploaded in this way from a GeoServer `.sld` file, you will need to use stack-internal URLs, e.g.:
+```
+<OnlineResource xlink:type="simple" xlink:href="http://<STACK_NAME>-geoserver:8080/geoserver/www/icons/myicon.png" />
```

Comment:
Might be worth using ```localhost``` here so new style is not needed when stack name changes
```suggestion
<OnlineResource xlink:type="simple" xlink:href="http://localhost:8080/geoserver/www/icons/myicon.png" />
```

Reply:
Thanks for the hint - a good suggestion. I did previously try a localhost address, but I guess I must have used the wrong port number... Anyway, I have committed this.